docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -266,8 +375,9 @@ To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck
 
 ## One-Time Selection
 
-To create a selector and immediately execute it without saving it,
-use the `select` method of the [test controller](../test-code-structure.md#test-controller).
+> Important! ***deprecated*** *Will be removed in next releases. Use [selectors](#creating-selectors) instead.*
```

Comment:
***Deprecated*** *Use [selectors](#creating-selectors) instead. The `select` method of the [test controller](../test-code-structure.md#test-controller) will be removed in future releases.*